cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,69 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
-    
+  const bookName = 'Speaking JavaScript';
+
+  beforeEach(() => {
```

Comment:
The beforeEach hook is empty, consider removing it or adding setup code. Consider implementation of baseUrl

Reply:
removed